src/Data/FileEmbed/Extra.hs

```diff
@@ -22,3 +42,43 @@ embedFileIfExists inputPath = do
           reportWarning $ "File " <> inputPath <> " not found"
           pure (LitE $ StringL "")
     (_, Nothing) -> fail "No filepath provided"
+
+-- | Like `embedFile`, but prints out debug logging when
+-- @FOSSA_DEBUG_EMBED_FILE@ is set.
+embedFileL :: FilePath -> Q Exp
+embedFileL fp = qAddDependentFile fp >> runIO embedWithLogging >>= bsToExp
```

Comment:
[question]: Why does this not just wrap around the `embedFile` function, like `embedFileIfExists`?  Why are we reimplementing?

Reply:
[naming]: Also, I prefer not to have ambiguous or unclear prefixes/suffixes on exported names.  If you're set on using "L", then please add a naming comment to the haddock.

Reply:
> Why are we reimplementing?

🤦🏼 Because I'm an idiot, apparently. I copied this from some 2 AM hacking code, I have no idea why I was opening up the internals. I'll update this to wrap.

> I prefer not to have ambiguous or unclear prefixes/suffixes on exported names

What name would you prefer? Some alternatives I considered:

- `embedFileWithLogging`
- `embedFile'`
- `embedFileDebug`

Reply:
> Because I'm an idiot
Big mood.
> Which name
`embedFile'` is my preferred.

I'd also like to make `embedFileIfExists` a wrapper around `embedFile'`, so we can get easy logging on those as well.